flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -486,7 +486,10 @@ class FMCMainDisplay extends BaseAirliners {
         SimVar.SetSimVarValue("L:A32NX_AP_CSTN_ALT", "feet", this.constraintAlt);
         SimVar.SetSimVarValue("L:A32NX_TO_CONFIG_NORMAL", "Bool", 0);
         SimVar.SetSimVarValue("L:A32NX_CABIN_READY", "Bool", 0);
+    }
 
+    connectedCallback() {
+        super.connectedCallback();
```

Comment:
this basically removes `SimVar.SetSimVarValue("K:A32NX.ATHR_RESET_DISABLE", "number", 1);` from `initVariables()`.

Why is that?

Reply:
![image](https://user-images.githubusercontent.com/15316958/128638008-195f3147-8023-4501-930b-c0c00583d870.png)

This is one of 2 K:Events that no longer works after SU5. The issue happens when they are called before fbw.wasm is compiled, so I simply attached it to connectedCallback() instead which is only called when the element is attached to the DOM.

The event is still initialised, it just occurs late enough that it does not trigger the issue.